src/Redistribution.sol

```diff
@@ -97,6 +97,23 @@ contract Redistribution is AccessControl, Pausable {
     // The reveal of the winner of the last round.
     Reveal public winner;
 
+    /**
+    * @dev Pause the contract. The contract is provably stopped by renouncing
+     the pauser role and the admin role after pausing, can only be called by the `PAUSER`
+     */
+    function pause() public {
+        require(hasRole(PAUSER_ROLE, msg.sender), "only pauser can pause");
```

Comment:
I recommend staying consistent with the messages in the `PriceOracle.sol`: "caller is not pauser". The same is below.